rfc6962-bis.xml

```diff
@@ -510,186 +596,170 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
+          <spanx style="verb">merkle_tree_head</spanx> is a <spanx style="verb">TransItem</spanx> structure that MUST be of type <spanx style="verb">tree_head</spanx> (see <xref target="tree_head"/>) and MUST have an empty <spanx style="verb">item_extensions</spanx> vector.
         </t>
-        <section title="Structure of the STH" anchor="sth_struct">
-          <figure>
-            <artwork>
-    enum {
-        v2(1), (255)
-    } TreeHeadVersion;
-
-    enum {
-        reserved(65535)
-    } SthExtensionType;
-
+      </section>
+      <section title="Merkle Consistency Proofs">
+        <figure>
+          <preamble>
+            To prepare a Merkle Consistency Proof for distribution to clients, the log produces a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">consistency_proof</spanx>, which in this version (v2) encapsulates a <spanx style="verb">ConsistencyProofDataV2</spanx> structure:
+          </preamble>
+          <artwork>
     struct {
-        SthExtensionType sth_extension_type;
-        opaque sth_extension_data&lt;0..2^16-1&gt;;
-    } SthExtension;
-
-    SthExtension SthExtensions&lt;0..2^16-1&gt;;</artwork>
-          </figure>
-          <t>
-            <spanx style="verb">sth_extension_type</spanx> identifies a single extension from the IANA registry in <xref target="sth_extension_types"/>.
-          </t>
-          <t>
-            The interpretation of the <spanx style="verb">sth_extension_data</spanx> field is determined solely by the value of the <spanx style="verb">sth_extension_type</spanx> field. Each document that registers a new <spanx style="verb">sth_extension_type</spanx> must describe how to interpret the corresponding <spanx style="verb">sth_extension_data</spanx>.
-          </t>
-          <t>
-            The <spanx style="verb">SthExtensions</spanx> type is a vector of 0 or more extensions. This vector MUST NOT include more than one extension with the same <spanx style="verb">sth_extension_type</spanx>. The extensions in the vector MUST be ordered by the value of the <spanx style="verb">sth_extension_type</spanx> field, smallest value first.
-          </t>
-          <figure>
-            <artwork>
+        LogID log_id;
+        uint64 tree_size_1;
+        uint64 tree_size_2;
+        NodeHash consistency_path&lt;1..2^8-1&gt;;
+    } ConsistencyProofDataV2;</artwork>
+        </figure>
+        <t>
+          <spanx style="verb">log_id</spanx> is this log's unique ID, encoded in an opaque vector as described in <xref target="log_id"/>.
+        </t>
+        <t>
+          <spanx style="verb">tree_size_1</spanx> is the size of the older tree.
+        </t>
+        <t>
+          <spanx style="verb">tree_size_2</spanx> is the size of the newer tree.
+        </t>
+        <t>
+          <spanx style="verb">consistency_path</spanx> is a vector of base64 encoded Merkle Tree nodes proving the consistency of two STHs.
```

Comment:
I would imagine base64 encoding is only necessary when transmitting data using JSON?
Furthermore, if it's base64-encoded, then wouldn't the field end up being bigger than HASH_SIZE?